src/api/usersRouter.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import { z } from 'zod';
+
+import { User, UserScheme } from './entities/User';
+
+export const usersRouter = express.Router();
+usersRouter.get('/', async (_req, res) => {
+  const users = await User.query();
+  return res.status(200).json(users);
+});
+
+usersRouter.post('/', async (req, res) => {
+  const data = UserScheme.parse(req.body);
+  const user = await User.query().insert(data);
+  res.status(201).json(user);
+});
+
+const emailParser = z.object({
+  email: z.string(),
+});
+
+usersRouter.get('/byEmail', async (req, res) => {
```

Comment:
Most APIs I've used put the field in the query-string - so a request might look something like `/api/users/?email=test@example.com`. Unless you're following a specific pattern that we might look at adopting, I think we're better off sticking with a standard pattern.

Reply:
I think you are right, updated